.github/workflows/e2e-web.yml

```
@@ -0,0 +1,114 @@
+          pip install supabase python-dotenv
+
+      - name: Reset + seed demo user
+        if: env.HAS_SERVICE_ROLE_KEY == 'true'
+        # Scope the service-role key to the one step that needs it — every
+        # other step (npm install, playwright run) has no business reading it.
+        env:
+          SUPABASE_URL: ${{ secrets.E2E_SUPABASE_URL }}
+          SUPABASE_SERVICE_ROLE_KEY: ${{ secrets.E2E_SUPABASE_SERVICE_ROLE_KEY }}
+        run: python scripts/seed_demo_user.py --reset
+
+      - name: Note: skipping seed (no service-role key)
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)